tap_zendesk/client.py

```diff
@@ -24,127 +30,184 @@
 
 class ZendeskStream(RESTStream):
     """Zendesk stream class."""
+    def __init__(self, tap, name=None, schema=None, path=None):
+        super().__init__(tap, name, schema, path)
+        self.last_request_time = 0
+        self.allow_redirects = False
+        self.min_remain_rate_limit = self.config.get("min_remain_rate_limit", None)
```

Comment:
Lets add the default value as 0 to the param definition.
```suggestion
        self.min_remain_rate_limit = self.config["min_remain_rate_limit"]
```